test/intern.js

```diff
@@ -12,6 +12,16 @@ define({
     host: 'localhost',
     port: 4444
   },
+  kibana: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 5620
+  },
+  elasticsearch: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 9220
```

Comment:
They're static.  The kibana port can be changed at https://github.com/elastic/kibana/blob/master/tasks/config/run.js, and the es port at https://github.com/elastic/kibana/blob/master/tasks/config/esvm.js.  I'm not sure where to make it live without coupling grunt tasks and test files.  I'll think about it a bit, open to any suggestions.